src/sailor/db/elasticsearch.lua

```diff
@@ -0,0 +1,132 @@
+-------------------------------------------------------------------------------------------
+-- elasticsearch.lua: DB module for connecting, querying and serching through elasticsearch
+-- This file is a part of Sailor project
+-- Copyright (c) 2016 Etiene Dalcol <dalcol@etiene.net>
+-- License: MIT
+-- http://sailorproject.org
+-------------------------------------------------------------------------------------------
+
+
+
+local elasticsearch = require "elasticsearch"
+local client = elasticsearch.client()
+local elastic = {}
+
+
+
+
+function elastic.getinfo()
+	local data, err = client:info()
+	if data==nil then
+		return err
+	else
+		return data
+
+	end
+end
+
+function elastic.index(indexq, typeq, idq, body)
+	local data, err = client:index{
+	index = indexq,
+	type = typeq,
+	id = idq,
+	body = body
+}
+	if data==nil then
+		return err
+	else
+		return data
+
+	end
+end
+
+function elastic.get(indexq, typeq, idq)
+	local data, err = client:get{
+  	index = indexq,
+  	type = typeq,
+  	id = idq
+}
+	if data==nil then
+		return err
+	else
+		return data
+	end
+end
```

Comment:
These wrappers are used from inside the elastic_model.lua( This is the model file for ES). I use these functions inside the model when an object is instantiated. Is this approach fine ?